docs-kr/network/nodes.html

```diff
@@ -206,6 +206,7 @@ <h3>옵션</h3>
       interpolation: false,  // only for image and circularImage shapes
       useImageSize: false,  // only for image and circularImage shapes
       useBorderWithImage: false  // only for image shape
+      useImageCoords: false  / only for image shape
```

Comment:
syntax error

Reply:
Strange, I'm sure tests ran fine.

I do get the following file-related errors though:
<img width="753" alt="Screen Shot 2020-06-18 at 16 29 49" src="https://user-images.githubusercontent.com/48869228/85063812-10dc6c00-b181-11ea-9d7c-2eca56768987.png">

Reply:
`npx npm-run-all clean build test`